README.md

```diff
@@ -115,6 +115,31 @@ The following are **beta** commands, you can enable them with with `DD_BETA_COMM
 #### `gate`
 - `evaluate`: Evaluate [Quality Gates](src/commands/gate) rules in Datadog. [📚](https://docs.datadoghq.com/quality_gates/)
 
+
+### FIPS support
+
+The `fips` option allow `datadog-ci` to use a FIPS cryptographic module provider, if the OpenSSL library installed on the host system provides it.
+`datadog-ci` cannot assert if such a provider is available, and won't throw any error in case the provider is not FIPS compliant.
```

Comment:
```suggestion
Note that `datadog-ci` cannot assert if such a provider is available, and won't throw any error in case the provider is not FIPS compliant.
```

Reply:
> in case the provider is not FIPS compliant

Did you mean FIPS validated?

Reply:
Yes, thanks for catching that! 🙇